nucypher/cli/commands/stake.py

```diff
@@ -874,6 +884,73 @@ def prolong(general_config, transacting_staker_options, config_file, force, lock
     paint_stakes(emitter=emitter, staker=STAKEHOLDER)
 
 
+@stake.command()
+@group_transacting_staker_options
+@option_config_file
+@option_force
+@group_general_config
+@click.option('--index-1', help="First index of stake to merge", type=click.INT)
+@click.option('--index-2', help="Second index of stake to merge", type=click.INT)
+def merge(general_config, transacting_staker_options, config_file, force, index_1, index_2):
+    """Merge two stakes into one."""
+
+    # Setup
+    emitter = setup_emitter(general_config)
+    STAKEHOLDER = transacting_staker_options.create_character(emitter, config_file)
+    action_period = STAKEHOLDER.staking_agent.get_current_period()
+    blockchain = transacting_staker_options.get_blockchain()
+
+    client_account, staking_address = select_client_account_for_staking(
+        emitter=emitter,
+        stakeholder=STAKEHOLDER,
+        staking_address=transacting_staker_options.staker_options.staking_address,
+        individual_allocation=STAKEHOLDER.individual_allocation,
+        force=force)
+
+    # Handle stakes selection
+    stake_1 = None
+    stake_2 = None
+
+    if index_1 is not None and index_2 is not None:
+        stake_1 = STAKEHOLDER.stakes[index_1]
+        stake_2 = STAKEHOLDER.stakes[index_2]
+    elif index_1 is not None:  # 0 is valid.
+        stake_1 = STAKEHOLDER.stakes[index_1]
+    elif index_2 is not None:
+        stake_2 = STAKEHOLDER.stakes[index_1]
```

Comment:
Can this block be condensed into the following?

```suggestion
    if index_1 is not None:  # 0 is valid.
        stake_1 = STAKEHOLDER.stakes[index_1]
    if index_2 is not None:
        stake_2 = STAKEHOLDER.stakes[index_2]
```

Since you check whether the relevant stake is `None` below and act accordingly.

Reply:
If rejecting this suggestion, note that in the original code used `index_1` for `stake_2` instead of `index_2`. We may need to fortify with a test.

Reply:
this was done on purpose - first stake can be chosen from all editable stakes but second one - editable + same final period so it's why there index_1 fro stake_2

Reply:
ohh, I was wrong, should be `stake_1 = STAKEHOLDER.stakes[index_2]`, I'll add test for that